SyncKotlin/README.md

```diff
@@ -0,0 +1,46 @@
+Fit sync sample
+===============
+
+The purpose of this sample is to show how to implement a periodic upload of data
+to Google Fit.
+
+Pre-requisites
+--------------
+
+- Android API Level >= 14
+- Android Build Tools v29
+- Android Support Repository
+- Register a Google Project with an Android client per getting started instructions
+  http://developers.google.com/fit/android/get-started
+
+Background
+----------
+
+Many fitness apps have the requirement to sync data to Google Fit on a regular
+basis (e.g. daily). This sample demonstrates how to achieve this using
+[WorkManager](https://developer.android.com/topic/libraries/architecture/workmanager).
+
+The sample also demonstrates:
+
+-   Allowing for user-initiated immediate syncs
+-   Handling transient errors with the upload, and retrying
+-   Recording failure and using [notifications](https://developer.android.com/guide/topics/ui/notifiers/notifications)
+    to alert the user.
+-   Turning off the synchronisation after retries are exhausted and the
+    sync process ultimately fails.
+
+In this sample, a toy database `StepsDatabase` is used as the source of data to
```

Comment:
Change 'toy' database with 'demo', 'local', or remove. Toy is a bit confusing to me, as I'm thinking about kid toys. :)

Reply:
Realized later this is a mock database. I would use 'mock' instead of toy...

In this sample, a mock/fake database `MockStepsDatabase` is used as the source of data to be uploaded to Fit, which serves as a placeholder for a real local or remote repository.